query/src/main/kotlin/com/linecorp/kotlinjdsl/query/clause/from/SimpleAssociator.kt

```diff
@@ -0,0 +1,69 @@
+package com.linecorp.kotlinjdsl.query.clause.from
+
+import com.linecorp.kotlinjdsl.query.spec.SimpleAssociatedJoinSpec
+import com.linecorp.kotlinjdsl.query.spec.AssociatedJoinSpec
+import com.linecorp.kotlinjdsl.query.spec.Froms
+import com.linecorp.kotlinjdsl.query.spec.JoinSpec
+import com.linecorp.kotlinjdsl.query.spec.expression.EntitySpec
+import java.util.*
+import javax.persistence.criteria.Path
+import javax.persistence.criteria.Root
+
+class SimpleAssociator(
+    fromEntity: EntitySpec<*>,
+    joins: Collection<JoinSpec<*>>,
```

Comment:
-- english
How about we change the parameter name to associate instead of join?

And we can fix the type as `SimpleAssociatedJoinSpec`.
-- korean
파라미터명을 join 대신에 associate으로 변경하는 것은 어떤가요?

그리고 타입을 `SimpleAssociatedJoinSpec`으로 고정할 수 있을 것 같습니다.

Reply:
applied

-- korean
반영했습니다